src/main/java/com/thevoxelbox/voxelsniper/brush/Brush.java

```diff
@@ -217,10 +218,15 @@ protected final int clampWorldHeight(int height) {
      * @param z Z coordinate
      * @return Type ID of Block at given coordinates in the world of the targeted Block.
      */
+    @Deprecated //This should probably be replaced with the LocationWrapper variant below
```

Comment:
```suggestion
     * @return Type ID of Block at given coordinates in the world of the targeted Block.
     * @deprecated This should probably be replaced with the LocationWrapper variant below
     */
    @Deprecated
```
Not too sure, I don't really see an issue in using the x/y/z.